tests/features/collection/collection.event.feature

```diff
@@ -0,0 +1,121 @@
+@api
+Feature: As a user of the website
+  In order to find my way around events
+  I need to be able to view collection events grouped properly.
+
+  Scenario: Events should be filterable by future/past/my filters.
+    Given users:
+      | Username       | E-mail                      |
+      | katerpillar    | kater.pillar@example.com    |
+      | trustysidekick | trusty.sidekick@example.com |
+    And collection:
+      | title | Fairy Tail |
+      | state | validated  |
+    And event content:
+      | title                     | collection | start date   | end date            | created    | state     | author         |
+      | Sweet Palm                | Fairy Tail | now -1 years | now -1 years +1 day | now -4 day | validated | katerpillar    |
+      | Melted Hairdresser        | Fairy Tail | now -1 day   | now +1 day          | now -3 day | validated | katerpillar    |
+      | Hot Air                   | Fairy Tail | now +2 day   | now +4 day          | now -2 day | validated | katerpillar    |
+      | Walking Unofficial Humans | Fairy Tail | now -3 day   | now -1 day          | now        | validated | trustysidekick |
+    # Technical: use a separate step to create an event associated to the anonymous user.
```

Comment:
It's not needed to use a separate step for this, just leave the author name empty and it will be associated with the anonymous user.